src/test/java/pages/PlayListPage.java

```diff
@@ -0,0 +1,76 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class PlayListPage extends BasePage {
+
+    public PlayListPage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    // Locators
+    By getCreateNewPlayListBtn = By.xpath("//i[@data-testid=\"sidebar-create-playlist-btn\"][@title=\"Create a new playlist\"][@class=\"fa fa-plus-circle create\"]");
+    By getNewPlayListBtn = By.xpath("//li[@data-testid=\"playlist-context-menu-create-simple\"][contains(text(), \"New Playlist\")]");
+    By getTitleNameField = By.xpath("//input [@placeholder=\"↵ to save\"][@name=\"name\"][@required=\"required\"]");
+    By getVerifyNotificationMsg = By.cssSelector("div.success.show");
+    By getPlayListToDelete = By.xpath("//a[contains(text(), 'TestPro Playlist')]");
+    By getPlayListToDeleteBtn = By.xpath("//button[@class=\"del btn-delete-playlist\"][@title=\"Delete this playlist\"]");
+    By getPlayListToDeleteOkBtn = By.xpath("//button[@class=\"ok\"][contains(text(), OK)]");
+    By getSearchSongField = By.cssSelector("input[type='search']");
+    By getViewAllSongs = By.cssSelector("button[data-test='view-all-songs-btn']");
+    By getFirstSong = By.xpath("//*[@id=\"songResultsWrapper\"]//table[@class=\"items\"]");
+    By getAddSongToBtn = By.xpath("//section[@id='songResultsWrapper']//button[@data-test='add-to-btn']");
+    By getChoosePlayList = By.xpath("//section[@id='songResultsWrapper']//li[@class=\"playlist\"][contains(text(), playListName)]");
+    By getChoosePlayListField = By.xpath("//section[@id='songResultsWrapper']//li[@class=\"playlist\"][contains(text(), playListName)]");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```